### PR TITLE
Fix equipments support and expedition requirements to 2022.04

### PR DESCRIPTION
**Pull request by OniJAM**
修复了舰队菜单里大发系装备图标的显示
更新远征成功条件，来源: https://wikiwiki.jp/kancolle/%E9%81%A0%E5%BE%81
https://bbs.nga.cn/read.php?tid=21866416
ps. 我没有使用 escortSpecialFleetCompos，因为里面的成功条件都可以和海域基础成功条件合并从而更简洁

### CODE REVIEW

**Comment by OniJAM** [comment]
新增大成功率计算支持（通常型、桶型、旗舰lv型）替换原有提示

**Comment by Javran** [comment]
版本號不要動, 其他沒問題.

**Review by Javran — CHANGES_REQUESTED** [review]
README.md 和 package.json裏的版本號不要改(README.md裏換成 `### Pending` 之類的). 我習慣publish的時候統一更新.

**Comment by Javran** [comment]
Thanks!

File: ui/requirement-viewer/ereq-list-group-item/gs-rate-drum-item.es [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  </div>) : null
+}
+
+const GSRateDrumCheck = props =>
+  props.result.extra.drum >= props.ereq.max ? props.result.extra.rate :
+  props.ereq.min === 0 ? (props.result.extra.rate - 20.2) :
+  props.result.extra.drum >= props.ereq.min ? (props.result.extra.rate - 35.35) : 0
+
+const GSRateDrumItem = props => {
+  props.result.sat = GSRateDrumCheck(props) >= 100
```
Comment (Javran) [inline ui/requirement-viewer/ereq-list-group-item/gs-rate-drum-item.es:25]: In hindsight I should have took a closer look on this - have you actually tested that this is working? Why the F are you setting props from UI?

**Pull request merged**